docs/proposals/old-timer.md

```diff
@@ -37,12 +37,12 @@ This made it unreliable for use by system components which need a historical
 view.
 
 Since we already persist metrics into a sink, it does not make sense for
-Heapster itself to persist long-term metrics to disk itself.  Instead, we can
+Heapster itself to persist long-term metrics to the disk itself.  Instead, we can
```

Comment:
this phrasing is not incorrect.  It's fine to say "persist things to disk".

Reply:
plus, in the earlier case, "itself" refers to Heapster, while in the later case, itself is ambiguous.